### PR TITLE
feat: remove websocket notification for Exchange

### LINKED ISSUE #860: Remove websocket notification for Exchange currency
The user is able to exchange currencies between their own accounts with the Exchnage button.
image

In the background the same outgoing-payments and incomin-paymnet is created as for Sending money.
While we can't influence for now the fees for this, we would like the user to NOT receive notification that they received money.
This means we need to add to metadata a flag maybe, that the outcoming and incoming payments are from exchange button, and then when we fire the MONEY_RECEIVED for incoming payments, in this case we DON'T fire the event.

### PR DESCRIPTION
**Pull request by rico191013**
### Context

- fixes #860

### CODE REVIEW

**Review by beniaminmunteanu — APPROVED** [review]

File: packages/wallet/backend/src/utils/helpers.ts [hunk 1]
```diff
@@ -79,3 +82,7 @@ export const prefixSomeObjectKeys = (
     }),
     {}
   )
+
+import NodeCache from 'node-cache'
+
+export const Node_Cache = new NodeCache({ stdTTL: 60 * 60 })
```
Comment (dragosp1011) [inline packages/wallet/backend/src/utils/helpers.ts:88]: maybe rename to NodeCacheInstance

File: packages/wallet/backend/src/rafiki/service.ts [hunk 2]
```diff
@@ -315,7 +317,8 @@ export class RafikiService implements IRafikiService {
     )
 
     const user = await this.deps.userService.getByWalletId(source_ewallet)
-    if (user)
+    const isExchange = Node_Cache.get(wh.data.payment.id)
```
Comment (dragosp1011) [inline packages/wallet/backend/src/rafiki/service.ts:320]: is the payment id equal with quote id?

File: packages/wallet/backend/src/quote/service.ts [hunk 3]
```diff
@@ -255,10 +259,14 @@ export class QuoteService implements IQuoteService {
       paymentPointerId: senderPpId,
       amount,
       description: 'Currency exchange.',
-      isReceive: false,
+      isReceive: true,
       receiver: receiverPpUrl
     })
 
+    //OUTCOME
+    Node_Cache.set(quote.id, quote)
```
Comment (dragosp1011) [inline packages/wallet/backend/src/quote/service.ts:267]: as the quote is not used and only helps to check if it is an exchange payment, cache it with a `true` value (or empty object if it requires the value to be an object)

File: packages/wallet/backend/src/rafiki/service.ts (hunk 2 above)
Comment (rico191013) [reply packages/wallet/backend/src/rafiki/service.ts:320]: It's a little weird, but according to logs and database records, the IDs of outgoing payments and quotes are the same.

**Review by dragosp1011 — APPROVED** [review]

**Pull request merged**